tests/Resources/ti.platform.addontest.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions'),
+	utilities = require('./utilities/utilities');
+
+describe('Titanium.Platform', function () {
+
+	it.ios('#openURL(url, callback)', function (finish) {
+		Ti.Platform.openURL("randomapp://", e => finish());
+	});
+
+	it.ios('#openURL(url, options, callback)', function (finish) {
+		Ti.Platform.openURL("randomapp://", {}, e => finish());
```

Comment:
<!--
  1 failure:  tests/Resources/t...
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.platform.addontest.js;
  Line: 20;
-->
- :no_entry_sign: tests/Resources/ti.platform.addontest.js line 20 – Strings must use singlequote. (quotes)
- :warning: tests/Resources/ti.platform.addontest.js line 20 – 'e' is defined but never used. Allowed unused args must match /^_.+/u. (no-unused-vars)